example_relay.conf

```diff
@@ -127,6 +127,16 @@ docker:
   # Default: 5m
   clean_interval: 5m
 
+  # Per container memory allocation (in megabytes)
+  # Environment variable: $RELAY_DOCKER_CONTAINER_MEMORY
```

Comment:
We'll need to add this to http://docs.operable.io/docs/relay-environment-variables